### ISSUE TITLE
Sorting could use some improvements

### ISSUE THREAD

**Issue by FunctionalHacker** [hirsivaja/plugin.video.areena#42]
I don't know if this is a limitation of the YLE API but right now because of the sorting of episodes, this plugin is basically unusable for watching any series with more than a couple episodes.

Steps to reproduce:

1. Do a series search for "itse valtiaat"
2. Try to find episode 1 of the series
3. You will notice that the episodes are not sorted by episode number, but alphabetically

**Comment by hirsivaja** [comment]
I agree, the current implementation is not very good. Partly because of limitations in the API, but still...

Seems like Vesa Toivonen has already implemented this:
https://github.com/vesatoivonen/plugin.video.areena/commit/20b561ac4a49a2fe14989a32e3e81e55824db535

@vesatoivonen Pull request would be appreciated

**Comment by vesatoivonen** [comment]
My current implementation is bad and i'm planning to fix that once kodi19 is released with it's transition to python3.
